src/pretix/plugins/sendmail/views.py

```diff
@@ -36,18 +39,35 @@ def form_valid(self, form):
         self.request.event.log_action('pretix.plugins.sendmail.sent', user=self.request.user, data=dict(
             form.cleaned_data))
 
+        tz = pytz.timezone(self.request.event.settings.timezone)
+
         failures = []
         for o in orders:
-            try:
-                mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                     None, self.request.event, locale=o.locale, order=o)
-            except SendMailException:
-                failures.append(o.email)
-
-        if failures:
-            messages.error(self.request, _('Failed to send mails to the following users: {}'.format(' '.join(failures))))
-        else:
-            messages.success(self.request, _('Your message has been queued to be sent to the selected users.'))
+            if self.request.POST.get("action") == "preview":
+                message = self.request.POST.get('message_0')
```

Comment:
Actually, the ``_0`` is appended because this is the field of the first language (I didn't realize this when I commented here yesterday). If you change your event settings and enable German as a second language, you'll see that you get two textareas and a ``_1`` message. Obviously, it would be perfect to show the preview in all languages (you can get the list at ``request.event.settings.locales``).

As you noticed, you can't call ``format`` on a ``LazyI18nString``, which is a pretix-specific representation of a string that exists in multiple languages (there is documentation on it [here](https://docs.pretix.eu/en/latest/development/implementation/i18n.html#usage)). You can easily convert it into a string for example by ``form.cleaned_data['message'].localize('en')`` (or ``form.cleaned_data['message'].localize(locale)`` in the general case).

Does that help? :)